packages/components/psammead-play-button/README.md

```diff
@@ -21,6 +21,7 @@ The `PlayButton` component renders a button with a 'video' or 'audio' icon and o
 | datetime       | string    | No       | `null`    | `'PT2M30S'`              |
 | type           | string    | No       | `'video'` | `'audio'`                |
 | className      | string    | No       | `null`    | `'bar'`                  |
+| guidanceMessage| string    | No       | `null`    | `'Contains strong language'`|
```

Comment:
```suggestion
| guidanceMessage| string    | No       | `null`    | `'Contains strong language.'`|
```

Reply:
I feel like we should enforce the `.` full stop whenever we can, because this is part of a long string , which has distinct segments. The word 'Play' will come directly after this.